src/comps/update/index.js

```diff
@@ -30,7 +30,21 @@ async function updateDb(data, dbname, round) {
     })
 }
 
+
+async function updateRewardsSummary(round) {
+    await db.promise().query(`
```

Comment:
Let's add a comment here:
```suggestion
    // aggregates the sum of "passive" and "curating" rewards for each "LP_addr" for a given "round" by getting the sum of `amt` from `rewards_info` and `passive_rewards_info` tables
    await db.promise().query(`
```